### PR TITLE
Feature/additional restrictions for editor

### PR DESCRIPTION
**Pull request by ralfger**
https://app.asana.com/0/1193966985516600/1197679322696199
https://app.asana.com/0/1193966985516600/1197679322696193
https://app.asana.com/0/1193966985516600/1197507425441572

### CODE REVIEW

File: frontend/schemaCMS/src/shared/components/blockElement/blockElement.component.js [hunk 1]
```diff
@@ -23,19 +23,21 @@ export const BlockElement = props => {
...
+              onChange={handleChange}
+              autoWidth
+              fullWidth
+              value={element.name}
+              customInputStyles={customInputStyles}
+              {...restFormikProps}
+            />
+          ) : (
+            element.name
+          )}
```
Comment (cpBurn) [inline frontend/schemaCMS/src/shared/components/blockElement/blockElement.component.js:39]: can we have `renderWhenAdmin`  in this component at least?

Comment (cpBurn) [reply frontend/schemaCMS/src/shared/components/blockElement/blockElement.component.js:39]: Or even `renderWhenTrueOtherwise`

Comment (ralfger) [reply frontend/schemaCMS/src/shared/components/blockElement/blockElement.component.js:39]: Oh, I forget to change here also. Thanks for cathing 👍 .

Comment (ralfger) [reply frontend/schemaCMS/src/shared/components/blockElement/blockElement.component.js]: Fixed :)

**Review by arkbuchholz — APPROVED** [review]

**Pull request merged**